client/src/components/Header/components/login.tsx

```diff
@@ -36,6 +37,11 @@ const Login = ({
       data-test-label={dataTestLabel}
       data-playwright-test-label='header-sign-in-button'
       href={href}
+      onClick={() => {
+        callGA({
+          event: 'sign_in'
+        });
+      }}
```

Comment:
I just want to point out this can fail, and you will not necessarily know. The `onclick` fires before navigating to the `href`.

So, provided that is the behaviour (analytics) you want, there is no issue.